src/flowchem/devices/knauer/azura_compact.py

```diff
@@ -375,6 +375,7 @@ async def infuse(self):
         await self._transmit_and_parse_reply(PUMP_ON)
         self._running = True
         logger.info("Pump started!")
+        return True   # todo: error
```

Comment:
Do you need a fwwdback of this method? Why does it need return True?
If you need this return, it`ll be better write -> bool, right?